pkgs/by-name/ff/ffts/package.nix

```
@@ -0,0 +1,42 @@
+{
+  stdenv,
+  lib,
+  cmake,
+  git,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ffts";
+  version = "0.7";
+
+  src = fetchFromGitHub {
+    owner = "ValveSoftware";
+    repo = "ffts";
+    rev = "2c8da4877588e288ff4cd550f14bec2dc7bf668c";
+    hash = "sha256-Cj0n7fwFAu6+3ojgczL0Unobdx/XzGNFvNVMXdyHXE4=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
+  ];
+  buildInputs = [ ];
+
+  cmakeFlags = [
+    "-DCURRENT_GIT_VERSION=${lib.substring 0 7 src.rev}"
+    "-DENABLE_SHARED=ON"
+    "-Wno-deprecated"
+  ];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
See https://github.com/NixOS/nixpkgs/issues/208242
You are already using `lib.maintainers` and `lib.platforms` :)

---

Indeed; fixed.